module-bsp/board/rt1051/bellpx/hal/battery_charger/BatteryCharger.cpp

```diff
@@ -64,6 +64,7 @@ namespace hal::battery
         void sendNotification(Events event);
         SOC fetchBatterySOC() const;
         void pollFuelGauge();
+        bool try_enable_charging();
```

Comment:
why the naming uses different convention? not tryEnableCharging

Reply:
Good point! Fixed